web-console/src/components/show-log.tsx

```diff
@@ -71,12 +73,39 @@ export class ShowLog extends React.Component<ShowLogProps, ShowLogState> {
     }
   }
 
+  private tail() {
+      this.getLogInfo().then(() => {
+        if (this.state.tail) {
+          if (this.log.current) {
+            this.log.current.scrollTo(0, this.log.current.scrollHeight); }
+          setTimeout(function() {
+            this.tail();
+          }.bind(this), 2000);
+        }
+      });
```

Comment:
oh my the indentation in this function is all messed up also you do not need to do `.bind(this)` you can just use `() => {}` syntax

Reply:
changed